.changeset/brave-rockets-invent.md

```diff
@@ -0,0 +1,8 @@
+---
+"@comet/admin-theme": minor
+---
+
+Add custom theme for LinearProgress
```

Comment:
I think something like this would make more sense to developers updating their application. 

```suggestion
Add custom default styling for LinearProgress
```

or:

```suggestion
Adjust the styling of LinearProgress to match the Comet CI
```